internal/dag/cache.go

```diff
@@ -203,6 +208,38 @@ func (kc *KubernetesCache) Insert(obj interface{}) bool {
 		}
 		kc.httpproxydelegations[m] = obj
 		return true
+	case *serviceapis.GatewayClass:
+		m := toMeta(obj)
+		if kc.gatewayclasses == nil {
+			kc.gatewayclasses = make(map[Meta]*serviceapis.GatewayClass)
+		}
+		kc.Infof("Adding GatewayClass %s in namespace %s", m.name, m.namespace)
```

Comment:
are these debug logs?

Reply:
Right now, they are the only visible effect that service-apis types are being added and removed. As soon as I can add in some actual behaviour, I plan on removing this - I will add a TODO.